samples/add-feature-collection-layer-from-table/src/main/java/com/esri/arcgismaps/sample/addfeaturecollectionlayerfromtable/components/AddFeatureCollectionLayerFromTableViewModel.kt

```diff
@@ -0,0 +1,220 @@
+            geometryType = GeometryType.Polyline,
+            spatialReference = SpatialReference.wgs84()
+        )
+
+        linesCollectionTable.renderer = SimpleRenderer(
+            symbol = SimpleLineSymbol(
+                style = SimpleLineSymbolStyle.Dash,
+                color = Color.green,
+                width = 3f
+            )
+        )
+
+        val lineGeometry = Polyline(points = listOf(
+            Point(x = -79.497238, y = 8.849289, spatialReference = SpatialReference.wgs84()),
+            Point(x = -80.035568, y = 9.432302, spatialReference = SpatialReference.wgs84())
+        ))
+
+        val lineAttributes = mapOf("Boundary" to "AManAPlanACanalPanama")
+        val lineFeature = linesCollectionTable.createFeature(lineAttributes, lineGeometry)
+        linesCollectionTable.addFeature(lineFeature).onFailure { messageDialogVM.showMessageDialog(it) }
+
+        return linesCollectionTable
+    }
+
+    /**
+     * Create an in-memory polygon FeatureCollectionTable with a fill renderer and a single polygon feature.
+     */
+    private suspend fun createPolygonsCollectionTable(): FeatureCollectionTable {
+        val areaField = Field(
+            fieldType = FieldType.Text,
+            name = "Area",
+            alias = "Area name",
+            length = 40,
+            isEditable = true,
+            isNullable = false
+        )
+
+        val polygonsCollectionTable = FeatureCollectionTable(
+            fields = listOf(areaField),
+            geometryType = GeometryType.Polygon,
+            spatialReference = SpatialReference.wgs84()
+        )
+
+        // Create a simple fill symbol using a blue outline and cyan diagonal cross fill
+        val outlineSymbol = SimpleLineSymbol(
+            style = SimpleLineSymbolStyle.Solid,
+            color = Color.blue,
+            width = 2f
+        )
+        val fillSymbol = SimpleFillSymbol(
+            style = SimpleFillSymbolStyle.DiagonalCross,
+            color = Color.cyan,
+            outline = outlineSymbol
+        )
+        polygonsCollectionTable.renderer = SimpleRenderer(symbol = fillSymbol)
+
+        val polygonGeometry = Polygon(points = listOf(
+            Point(x = -79.497238, y = 8.849289, spatialReference = SpatialReference.wgs84()),
+            Point(x = -79.337936, y = 8.638903, spatialReference = SpatialReference.wgs84()),
+            Point(x = -79.11409, y = 8.895422, spatialReference = SpatialReference.wgs84())
+        ))
+
+        val polygonAttributes = mapOf("Area" to "Restricted area")
+        val polygonFeature = polygonsCollectionTable.createFeature(polygonAttributes, polygonGeometry)
+        polygonsCollectionTable.addFeature(polygonFeature).onFailure { messageDialogVM.showMessageDialog(it) }
+
+        return polygonsCollectionTable
+    }
+}
+
+/**
+ * Define a blue color.
+ */
+private val Color.Companion.blue: Color
+    get() {
+        return fromRgba(r = 0, g = 0, b = 255, a = 255)
+    }
```

Comment:
nitpick...no end of line at end of file
```suggestion
    }
    
```